src/Nut/DebugServiceRegistration.php

```diff
@@ -30,27 +34,43 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $this->displayRegistration($output);
+        $this->displayRegistration($input, $output);
     }
 
     /**
+     * @param InputInterface  $input
      * @param OutputInterface $output
      */
-    private function displayRegistration(OutputInterface $output)
+    private function displayRegistration(InputInterface $input, OutputInterface $output)
     {
         $table = new Table($output);
+        $rightAligned = new TableStyle();
+        $rightAligned->setPadType(STR_PAD_LEFT);
+
         $table->setHeaders([
-            '#',
-            'Provider',
+            [new TableCell('System service provider registration boot order', ['colspan' => 2])],
```

Comment:
How about just `Service Providers`? You have a lot of extra words in there that don't mean anything :)